src/components/icon/icon.spec.tsx

```diff
@@ -470,6 +470,18 @@ describe("Icon component", () => {
       );
     });
 
+    it("does not set an aria label if hideAriaLabel is true", () => {
+      const wrapper = mount(
+        <Icon type="home" hideAriaLabel ariaLabel="home" />
+      );
+
+      expect(wrapper.find(StyledIcon).props()).toEqual(
```

Comment:
I went for this because it's the same format that is used to test the `aria-label` further up in the tests so I wanted to keep things congruent. I am happy to change it your way though.

Reply:
fair enough, I'd missed that. Personally I'd still write this test what I think is the better way, and maybe even change the other one - but it doesn't really matter. (The one scenario that does matter is if somehow the icon ended up with an aria-label taht was different from the one passed in the prop - then your test would pass while my suggestion would fail. But that's such an out-there situation I'm not going to try to use this as a serious argument in favour of mine 😂 )

Reply:
Changed the unit tests as per the other suggested changes so this was kind of redundant now.